anchor/client/src/cli.rs

```diff
@@ -329,6 +329,14 @@ pub struct Anchor {
         help_heading = FLAG_HEADER
     )]
     help: Option<bool>,
+    #[clap(
+        long,
+        global = true,
+        value_delimiter = ',',
+        help = "One or more comma-delimited base64-encoded ENR's to bootstrap the p2p network",
+        display_order = 0
```

Comment:
We can declare the default boot nodes somewhere (like `anchor/network/src/config.rs`) with:
```rust
const DEFAULT_BOOTNODES: [&'static str; 3] = ["ENR1", "ENR2", "ENR3"];
```
and then import it and use it here: 
```suggestion
        display_order = 0
        default_values_t = DEFAULT_BOOTNODES.into_iter().map(Into::into)
```

Reply:
It seems LH defines them in `yaml` files https://github.com/sigp/lighthouse/blob/stable/common/eth2_network_config/built_in_network_configs/holesky/boot_enr.yaml

Reply:
yeah can also be, but then we should get them by default, if you prefer it can be done in a subsequent PR

Reply:
Yes, it's probably better in another PR.